packages/react-core/src/next/components/Wizard/WizardToggle.tsx

```diff
@@ -34,25 +32,24 @@ export interface WizardToggleProps {
 
 export const WizardToggle = ({
   steps,
-  currentStep,
+  activeStep,
   footer,
   nav,
   isNavExpanded,
   toggleNavExpanded,
-  hasUnmountedSteps = true,
   'aria-label': ariaLabel = 'Wizard toggle'
 }: WizardToggleProps) => {
-  const isActiveSubStep = isWizardSubStep(currentStep);
+  const isActiveSubStep = isWizardSubStep(activeStep);
   const nonSubSteps = steps.filter(step => !isWizardSubStep(step));
   const wizardToggleIndex =
     nonSubSteps.indexOf(
-      isWizardSubStep(currentStep) ? steps.find(step => step.id === currentStep.parentId) : currentStep
+      isWizardSubStep(activeStep) ? steps.find(step => step.id === activeStep.parentId) : activeStep
     ) + 1;
 
   const handleKeyClicks = React.useCallback(
     (event: KeyboardEvent): void => {
       if (isNavExpanded && event.key === KeyTypes.Escape) {
-        toggleNavExpanded();
+        toggleNavExpanded?.();
```

Comment:
I've never seen someone use this syntax like that! cool!

Reply:
+1! Optional chaining is sweeeet